.github/workflows/cicd-main.yml

```diff
@@ -5223,7 +5223,6 @@ jobs:
       - L0_Unit_Tests_GPU_Hydra
       #- OPTIONAL_L0_Unit_Tests_GPU_Lightning
       - L0_Unit_Tests_GPU_Others
-      
```

Comment:
This line was deliberately here to separate CPU from GPU for better readability/awareness

Reply:
So can we revert it back please if you dont mind, at least it helps me in readability :)